tests/functional/search-non-auth.js

```diff
@@ -36,7 +36,7 @@ registerSuite("Search (non-auth)", {
       return FunctionalHelpers.openPage(
         this,
         url("/issues"),
-        "[data-remotename=browser-android"
+        "[data-remotename=browser-android]"
```

Comment:
🔍 weird.

Reply:
It was wrong, right?

Reply:
(yeah, i'm mostly concerned that we didn't know until you saw it 😄 )